<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="windowTitle">
   <string>Frontend settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_show_fallbacks">
     <property name="text">
      <string>Show fallbacks on empty results</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_show_fallbacks</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_show_fallbacks">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Automatically enter the fallbacks mode if a query has no matches.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_history_search">
     <property name="text">
      <string>Enable history search</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_history_search</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_history_search">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use the current input text to filter the input history.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_clearOnHide">
     <property name="text">
      <string>Clear input line on hide</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_clearOnHide</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_clearOnHide">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear the input line when the window is hidden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_onTop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Always on top</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_onTop</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_onTop">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Let Albert stay on top of all top level windows.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_quit_on_close">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Quit on close</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_quit_on_close</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_quit_on_close">
     <property name="toolTip">
      <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quit the application when closing Albert. Default behavior is to hide the window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_hideOnFocusOut">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Hide on focus out</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_hideOnFocusOut</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_hideOnFocusOut">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hide Albert if the window lost focus.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_center">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Center on current display</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_center</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBox_center">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Center the main window on the screen containing the mouse pointer on show.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_style">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Theme</string>
     </property>
     <property name="buddy">
      <cstring>comboBox_themes</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_themes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the color theme of the window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-size:9pt; color:#818181;&quot;&gt;If you created a cool theme &lt;/span&gt;&lt;a href=&quot;https://github.com/orgs/albertlauncher/discussions/1147&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#094fd1;&quot;&gt;share it&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:9pt; color:#818181;&quot;&gt; with other users. The best themes will go upstream and be shipped with Albert.&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_results">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Displayed items</string>
     </property>
     <property name="buddy">
      <cstring>spinBox_results</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="spinBox_results">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_system_shadow">
     <property name="text">
      <string>Display system shadow</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_system_shadow</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="checkBox_system_shadow">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tell the system to draw a shadow around the window. Note that on some systems this may look pretty ugly. Use the client shadow in this case.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_client_shadow">
     <property name="text">
      <string>Display client shadow</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_client_shadow</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="checkBox_client_shadow">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Draw a shadow on the client side.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_icons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Display icons</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_icons</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="checkBox_icons">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Display icons provided by the items.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_scrollbar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Display scrollbar</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_scrollbar</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="checkBox_scrollbar">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Displays a scrollbar in the results list.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
